sys-devel/llvm/llvm16-16.0.6.recipe

```diff
@@ -0,0 +1,721 @@
+	devel:libLLVMInterfaceStub$secondaryArchSuffix
+	devel:libLLVMInterpreter$secondaryArchSuffix
+	devel:libLLVMIRPrinter$secondaryArchSuffix
+	devel:libLLVMJITLink$secondaryArchSuffix
+	devel:libLLVMLTO$secondaryArchSuffix
+	devel:libLLVMLanaiAsmParser$secondaryArchSuffix
+	devel:libLLVMLanaiCodeGen$secondaryArchSuffix
+	devel:libLLVMLanaiDesc$secondaryArchSuffix
+	devel:libLLVMLanaiDisassembler$secondaryArchSuffix
+	devel:libLLVMLanaiInfo$secondaryArchSuffix
+	devel:libLLVMLibDriver$secondaryArchSuffix
+	devel:libLLVMLineEditor$secondaryArchSuffix
+	devel:libLLVMLinker$secondaryArchSuffix
+	devel:libLLVMMC$secondaryArchSuffix
+	devel:libLLVMMCA$secondaryArchSuffix
+	devel:libLLVMMCACustomBehaviourAMDGPU$secondaryArchSuffix
+	devel:libLLVMMCDisassembler$secondaryArchSuffix
+	devel:libLLVMMCJIT$secondaryArchSuffix
+	devel:libLLVMMCParser$secondaryArchSuffix
+	devel:libLLVMMIRParser$secondaryArchSuffix
+	devel:libLLVMMSP430AsmParser$secondaryArchSuffix
+	devel:libLLVMMSP430CodeGen$secondaryArchSuffix
+	devel:libLLVMMSP430Desc$secondaryArchSuffix
+	devel:libLLVMMSP430Disassembler$secondaryArchSuffix
+	devel:libLLVMMSP430Info$secondaryArchSuffix
+	devel:libLLVMMipsAsmParser$secondaryArchSuffix
+	devel:libLLVMMipsCodeGen$secondaryArchSuffix
+	devel:libLLVMMipsDesc$secondaryArchSuffix
+	devel:libLLVMMipsDisassembler$secondaryArchSuffix
+	devel:libLLVMMipsInfo$secondaryArchSuffix
+	devel:libLLVMNVPTXCodeGen$secondaryArchSuffix
+	devel:libLLVMNVPTXDesc$secondaryArchSuffix
+	devel:libLLVMNVPTXInfo$secondaryArchSuffix
+	devel:libLLVMObjCARCOpts$secondaryArchSuffix
+	devel:libLLVMObjCopy$secondaryArchSuffix
+	devel:libLLVMObject$secondaryArchSuffix
+	devel:libLLVMObjectYAML$secondaryArchSuffix
+	devel:libLLVMOption$secondaryArchSuffix
+	devel:libLLVMOrcJIT$secondaryArchSuffix
+	devel:libLLVMOrcShared$secondaryArchSuffix
+	devel:libLLVMOrcTargetProcess$secondaryArchSuffix
+	devel:libLLVMPasses$secondaryArchSuffix
+	devel:libLLVMPowerPCAsmParser$secondaryArchSuffix
+	devel:libLLVMPowerPCCodeGen$secondaryArchSuffix
+	devel:libLLVMPowerPCDesc$secondaryArchSuffix
+	devel:libLLVMPowerPCDisassembler$secondaryArchSuffix
+	devel:libLLVMPowerPCInfo$secondaryArchSuffix
+	devel:libLLVMProfileData$secondaryArchSuffix
+	devel:libLLVMRISCVAsmParser$secondaryArchSuffix
+	devel:libLLVMRISCVCodeGen$secondaryArchSuffix
+	devel:libLLVMRISCVDesc$secondaryArchSuffix
+	devel:libLLVMRISCVDisassembler$secondaryArchSuffix
+	devel:libLLVMRISCVInfo$secondaryArchSuffix
+	devel:libLLVMRISCVTargetMCA$secondaryArchSuffix
+	devel:libLLVMRISCVUtils$secondaryArchSuffix
+	devel:libLLVMRemarks$secondaryArchSuffix
+	devel:libLLVMRuntimeDyld$secondaryArchSuffix
+	devel:libLLVMScalarOpts$secondaryArchSuffix
+	devel:libLLVMSelectionDAG$secondaryArchSuffix
+	devel:libLLVMSparcAsmParser$secondaryArchSuffix
+	devel:libLLVMSparcCodeGen$secondaryArchSuffix
+	devel:libLLVMSparcDesc$secondaryArchSuffix
+	devel:libLLVMSparcDisassembler$secondaryArchSuffix
+	devel:libLLVMSparcInfo$secondaryArchSuffix
```

Comment:
Are these really provided?

Reply:
Valid comment, there were a bunch of spurious provides. Cleaning it up.